torchvision/datasets/utils.py

```diff
@@ -7,11 +7,28 @@
 from typing import Any, Callable, List, Iterable, Optional, TypeVar
 from urllib.parse import urlparse
 import zipfile
+import urllib
+import urllib.request
+import urllib.error
 
 import torch
 from torch.utils.model_zoo import tqdm
 
 
+USER_AGENT = "pytorch/vision"
```

Comment:
Following will make it customizable and overridable(along with previous change to import defitions
```suggestion
USER_AGENT = os.getenv("TORCHVISION_USER_AGENT ", f"pytorch-{torch.__version__}/vision-{__vision_version__}"
```